packages/apps/tools/src/constants/kadena.ts

```diff
@@ -1,12 +1,11 @@
-import type { ChainwebNetworkId } from '@kadena/chainweb-node-client';
-
 import { env } from '@/utils/env';
 
-export type Network = Exclude<ChainwebNetworkId, 'development'>;
+export type DefinedNetwork = 'mainnet01' | 'testnet04';
```

Comment:
Out of curiosity; why can't we use `ChainwebNetworkId` anymore?

Reply:
I think @alber70g would know the answer to this :)